storage/nfs/tc515688/pv1.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-one
+spec:
+  capacity:
+    storage: 2Gi
+  accessModes:
+    - ReadWriteOnce
+  nfs:
+    path: /sharezyp
+    server: 10.14.6.144
```

Comment:
@jhou1 Could you help review?I'm not sure if our nfs server could be shared in this repo.